libs/checkpoint-postgres/langgraph/checkpoint/postgres/base.py

```diff
@@ -1,21 +1,38 @@
 from __future__ import annotations
 
 import random
+import warnings
 from collections.abc import Sequence
+from importlib.metadata import version as get_version
 from typing import Any, Optional, cast
 
 from langchain_core.runnables import RunnableConfig
 from langgraph.checkpoint.base import (
     WRITES_IDX_MAP,
     BaseCheckpointSaver,
     ChannelVersions,
+    CheckpointMetadata,
     get_checkpoint_id,
+    get_checkpoint_metadata,
 )
 from langgraph.checkpoint.serde.types import TASKS
 from psycopg.types.json import Jsonb
 
 MetadataInput = Optional[dict[str, Any]]
 
+try:
+    major, minor = get_version("langgraph").split(".")[:2]
+    if int(major) == 0 and int(minor) < 5:
+        warnings.warn(
+            "LangGraph versions < 0.5.x may have compatibility issues with this version of checkpoint-postgres. "
```

Comment:
This should be worded more like "You're using incompatible versions of langgraph and langgraph-checkpoint-postgres. ..."